.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Force shell scripts to always have LF endings, even on Windows.
+*.sh text eol=lf
```

Comment:
I don't think .gitattributes are applied for evergreen patches, so I don't think this won't work when adding a file in a patch build, fyi.